### PR TITLE
bpo-46878: Purge 'non-standard' from sqlite3 docstrings

### PR DESCRIPTION
**Pull request by erlend-aasland**
https://bugs.python.org/issue46878

### CODE REVIEW

File: Modules/_sqlite/module.c [hunk 1]
```diff
@@ -111,12 +111,15 @@ _sqlite3.enable_shared_cache as pysqlite_enable_shared_cache
 
 Enable or disable shared cache mode for the calling thread.
 
-Experimental/Non-standard.
+This method is deprecated and will be removed in Python 3.12.
```
Comment (JelleZijlstra) [inline Modules/_sqlite/module.c:114]: This carries on changes from https://github.com/python/cpython/commit/ddb5e11683c8db9d2095f2f9406701443c4dc9b3 (no change requested, just noting why this change makes sense)

**Review by JelleZijlstra — APPROVED** [review]
Looks good, thanks.

Do you think we should backport this change? I imagine it will make future backports easier if we keep the docstrings similar.

File: Modules/_sqlite/module.c (hunk 1 above)
Comment (erlend-aasland) [reply Modules/_sqlite/module.c:114]: Yes, and that PR _should_ (IMO) also have changed the docstring.

**Review by gpshead — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
Going to merge this change soon (cc @gvanrossum).
